assets/image/chart.svg

```diff
@@ -0,0 +1,161 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1280" height="960">\n<path  d="M 0 0
```

Comment:
I think it's name "chart" is too general.
It should be "QPS of Agents vs Recall", "QPS vs Recall" or something else.
And it should be placed under /assets/image/charts directory.